### PR TITLE
Mark plugins which have CakePHP 4 compatible releases

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,17 +194,17 @@ Additional lists you might find useful:
 ## I18n
 *Plugins for I18n (Internationalization) and L10n (Localization).*
 
-- [ADmad/I18n plugin](https://github.com/ADmad/cakephp-i18n) - A plugin with I18n related tools.
+- :strawberry: [ADmad/I18n plugin](https://github.com/ADmad/cakephp-i18n) - A plugin with I18n related tools.
 - [Localized plugin](https://github.com/cakephp/localized) - Localized validation and ready-to-use translation PO files.
-- [ShadowTranslate plugin](https://github.com/AD7six/cakephp-shadow-translate) - A plugin with shadow table based replacement for core's Translate behavior.
+- :strawberry: [ShadowTranslate plugin](https://github.com/AD7six/cakephp-shadow-translate) - A plugin with shadow table based replacement for core's Translate behavior.
```
Comment (garas) [inline README.md]: Plugin itself has no 4.x support.

Add information about how to upgrade to core version in the plugin repository or link to the book.

Comment (ADmad) [reply README.md]: Yup I am going to update the plugin's readme to point to relevent manual page for 4.x.

**Comment by dereuromark** [comment]
Releases = at least beta tag? no dev-master branch only, right?

**Comment by ADmad** [comment]
@dereuromark Yup

**Comment by dereuromark** [comment]
Then you can add 
- https://github.com/dereuromark/cakephp-ide-helper
- https://github.com/dereuromark/cakephp-flash

File: README.md (hunk 1 above)
Comment (dereuromark) [reply README.md]: Does that mean it is actually somewhat deprecated as plugin now?

**Comment by josbeir** [comment]
You can add (made a release for 4.x)

- https://github.com/josbeir/cakephp-filesystem

**Pull request merged**